contracts/StablecoinConverter.sol

```diff
@@ -63,17 +63,18 @@ contract StablecoinConverter is EpochTokenLocker {
         uint16 sellToken,
         bool isSellOrder,
         uint32 validUntil,
-        uint128 buyAmount,
-        uint128 sellAmount
+        uint128 priceNominator,
+        uint128 priceDenominator
```

Comment:
The user stills provide the buy amount and sell amount, I wouldn't change the name of the signature

Also, it would be more explicit and easy to understand when you read the initial values of the order:
```
            priceNominator: buyAmount,
            priceDenominator: sellAmount,
            remainingAmount: sellAmount
```